ace.d.ts

```diff
@@ -152,7 +152,7 @@ export namespace Ace {
   }
 
   export interface EditSessionOptions {
-    wrap: string | number;
+    wrap: boolean | number;
```

Comment:
wrap also [supports](https://github.com/ajaxorg/ace/blob/master/lib/ace/edit_session.js#L2460) "off" | "free" | "printmargin" or a string containing a number?

Is there useful way to write this type, or should we use `boolean | number | string`?

Reply:
Yes, we can strictly type exact that. Let me update it

Reply:
What solution did you have in mind?